tools/downloader/common.py

```diff
@@ -30,6 +30,7 @@
 
 # make sure to update the documentation if you modify these
 KNOWN_FRAMEWORKS = {'caffe', 'dldt', 'mxnet', 'pytorch', 'tf'}
+KNOWN_ONNX_SUPPORT = {'pytorch'}
```

Comment:
The comment above doesn't apply to this variable, so please move it out of this block.

Reply:
Disagree. If you add new framework, which is supported via conversion to ONNX, you also need to update documentation, for example contribution guide (in future).